### ISSUE TITLE
MD050: A series of underscores or asterisks are seen as an attempt to use emphasis

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#489]
### rumdl version

0.1.40

### Installation method

mise

### Bug description

Rule MD050 (https://github.com/rvben/rumdl/blob/59433519e67ef0f5071bc6cdcd38c898009bc726/docs/md050.md) is used to ensure that a single type of bold marker is used among the two options available.

At the moment, the rule interprets `_____` (five underscores) as an attempt to emphasise  something, but I don't think this should be considered this way.

### Steps to reproduce

1. Create a Markdown file with this content:
   ```markdown
   ---
   title: Level 1 heading
   ---

   This is a series of underscores: _____
   ```
1. Run `rumdl check` on the Markdown file with default options.

### Expected behavior

I'd expect `rumdl` to see this as a series of underscores and not as trying to emphasise a `_` character.

### Actual behavior

`rumdl` returns this message:

```shell
test.md:5:34: [MD050] Strong emphasis should use ** instead of __ [*]
```

### Configuration

None, use default configuration.

### Operating system

macOS

### Additional context

If you set underscores as your preferred bold marker in configuration, then this line:

```markdown
This is a series of asterisks: *****
```

will result in this output:

```shell
test.md:5:32: [MD050] Strong emphasis should use __ instead of ** [*]
```

**Comment by rvben** [comment]
Fixed in [v0.1.41](https://github.com/rvben/rumdl/releases/tag/v0.1.41). Sequences of literal underscores or asterisks are no longer mistakenly treated as emphasis. Thanks for reporting @eread!

**Comment by eread** [comment]
> Fixed in [v0.1.41](https://github.com/rvben/rumdl/releases/tag/v0.1.41). Sequences of literal underscores or asterisks are no longer mistakenly treated as emphasis. Thanks for reporting [@eread](https://github.com/eread)!

Thanks @rvben. This is fixed!
